ActiveLogin.Authentication.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.168
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28606.126
```

Comment:
I'd roll back the VS version change for now, we don't run VS2019 on our build servers for example.